exampleCourse/questions/element/matching/question.html

```diff
@@ -1,21 +1,113 @@
+        plus random distractors.
     </p>
 </pl-question-panel>
     
-<pl-matching answers-name="capitals" number-statements="3" number-options="5" blank="true">
+<pl-matching answers-name="ex3" number-statements="3" number-options="5">
     <pl-statement match="Ottawa">Canada</pl-statement>
-    <pl-statement match="d.c.">United States</pl-statement>
+    <pl-statement match="Washington, D.C.">United States</pl-statement>
     <pl-statement match="Mexico City">Mexico</pl-statement>
     <pl-statement match="Paris">France</pl-statement>
     
-    <pl-option>Ottawa</pl-option>
-    <pl-option name="d.c.">Washington, D.C.</pl-option>
-    <pl-option>Mexico City</pl-option>
     <pl-option>Tijuana</pl-option>
-    <pl-option>Calgary</pl-option>
     <pl-option>Toronto</pl-option>
     <pl-option>New York City</pl-option>
-    <pl-option>Paris</pl-option>
+</pl-matching>
+
+<pl-question-panel>
+    <p>
+        Example 4: if <code>number-options</code> is smaller than the amount of options
+        required to match the sampled statements, None of the Above will be added.
+    </p>
+</pl-question-panel>
+    
+<pl-matching answers-name="ex4" number-statements="3" number-options="2">
+    <pl-statement match="Ottawa">Canada</pl-statement>
+    <pl-statement match="Washington, D.C.">United States</pl-statement>
+    <pl-statement match="Mexico City">Mexico</pl-statement>
+    <pl-statement match="Paris">France</pl-statement>
+    
+    <pl-option>Tijuana</pl-option>
+    <pl-option>Toronto</pl-option>
+    <pl-option>New York City</pl-option>
+</pl-matching>
+
+<pl-question-panel>
+    <p>
+        Example 5: the <code>counter-type</code> can be changed to lower-alpha, upper-alpha, or
+        decimal.
+    </p>
+</pl-question-panel>
+    
+<pl-matching answers-name="ex5" counter-type="decimal" number-statements="3" number-options="3">
+    <pl-statement match="Ottawa">Canada</pl-statement>
+    <pl-statement match="Washington, D.C.">United States</pl-statement>
+    <pl-statement match="Mexico City">Mexico</pl-statement>
+    <pl-statement match="Paris">France</pl-statement>
+    
+    <pl-option>Tijuana</pl-option>
+    <pl-option>Toronto</pl-option>
+    <pl-option>New York City</pl-option>
+</pl-matching>
+
+<pl-question-panel>
+    <p>
+        Example 6: options can be added manually if they require more than simple text.
+        All PrairieLearn and HTML elements are supported in options and statements.
+    </p>
+</pl-question-panel>
+    
+<pl-matching answers-name="ex6">
+    <pl-statement match="USA">United States</pl-statement>
+    <pl-statement match="France">France</pl-statement>
+    
+    <pl-option name="USA"><img src="https://upload.wikimedia.org/wikipedia/en/thumb/a/a4/Flag_of_the_United_States.svg/320px-Flag_of_the_United_States.svg.png"/></pl-option>
+    <pl-option name="France"><img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/c3/Flag_of_France.svg/320px-Flag_of_France.svg.png"/></pl-option>
```

Comment:
This is a great example of using custom HTML, but I don't like relying on external resources. Can you copy these images to the `questionFilesCourse` folder? Alternatively, can you make a question that uses the files in `exampleCourse/clientFilesCourse/student_images`? Questions like `demo/studentFaces` or `demo/studentNames` may be a starting point for what to expect.